src/AnimationManager.ts

```diff
@@ -141,20 +141,38 @@ export class AnimationManager {
 			const info: AnimationParam = this._animateParam;
 			const self = this;
 			let prevPos = info.depaPos;
+			const directions = map(prevPos, (value, key) => {
+				return value <= info.destPos[key] ? 1 : -1;
+			});
+			let prevTime = new Date().getTime();
+			info.startTime = prevTime;
 
-			info.startTime = new Date().getTime();
 			(function loop() {
 				self._raf = null;
-				const easingPer = self.easing((new Date().getTime() - info.startTime) / param.duration);
-				const toPos: Axis = map(info.depaPos, (pos, key) => pos + info.delta[key] * easingPer);
-				const isCanceled = !self.em.triggerChange(toPos, false, prevPos);
+				const currentTime = new Date().getTime();
+				const prevEasingPer = self.easing((prevTime - info.startTime) / param.duration);
+				const easingPer = self.easing((currentTime - info.startTime) / param.duration);
+				let toPos: Axis = map(prevPos, (pos, key) => pos + info.delta[key] * (easingPer - prevEasingPer));
+				const isCanceled = !self.em.triggerChange(toPos, false, mapToFixed(prevPos));
 
-				prevPos = map(toPos, v => toFixed(v));
+				toPos = self.axm.map(toPos, (pos, options, key) => {
```

Comment:
@daybrush Is `options` different from options on constructor? Is it changed by user setting, right?